src/test/java/com/github/jcustenborder/kafka/connect/transform/xml/FromXmlTest.java

```diff
@@ -63,24 +68,14 @@ void apply() throws IOException {
 
     ConnectRecord record = this.transform.apply(inputRecord);
 
-    Schema schema = record.valueSchema();
```

Comment:
I had to revert this because you are removing xpath logic feature (bc you based it on the master branch). can you reapply your changes on top of custombuild and add another change?